### ISSUE TITLE
Modify: HP:0032790(Focal impaired awareness behavior arrest seizure)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5658]
HP:0032790(Focal impaired awareness behavior arrest seizure)\
Parent: HP:0011146(Dialeptic seizure)\
Add missing relation: Add

**Comment by pnrobinson** [comment]
@DLewisSmith
Could I ask for clarification. Dialeptic seizure is defined as
A dialeptic seizure is a type of seizure characterised predominantly by reduced responsiveness or awareness and with subsequent at least partial amnesia of the event.

Therefore, we make Focal impaired awareness behavior arrest seizure a child of this term instead of Focal non-motor impaired awareness seizure HP:0032716 (https://hpo.jax.org/app/browse/term/HP:0032716).

**Comment by DLewisSmith** [comment]
I would leave "Dialeptic" as a parent of "Focal impaired awareness behavior arrest seizure" and "Generalized non-motor (absence) seizure" but not of "Focal non-motor impaired awareness seizure".

I thought about this one a bit, mainly because Luders' original definition of a dialeptic seizure implies that all complex partial seizures would be a subtype of dialeptic seizure (Luders H, Acharya J, Baumgartner C, Benbadis S, Bleasel A, Burgess R, et al. Semiological seizure classification. Epilepsia 1998; 39(9): 1006-13). However, we interpret their definition of "dialeptic" 

“Dialeptic” seizures have as their **main** ictal manifestations an alteration of consciousness that is independent of ictal EEG manifestations.


Under modern ILAE concepts, not all focal impaired awareness seizures may need to be called dialeptic as the ILAE classification allows clinicians to  use focal symptoms such as deja-vu and hallucinations occurring prior to loss of awareness to establish that the seizure is focal in onset.

This is still contentious as there are opposing schools of thought. We may end up modifying the name of this term to 'Purely dialectic seizure' or similar depending on feedback from the community. However, the concept of a seizure in which the patient freezes without any motor signs, and with our reporting non-motor symptoms and cannot recall the event afterwards independent of onset needs to remain.

**Comment by pnrobinson** [comment]
addressed by https://github.com/obophenotype/human-phenotype-ontology/issues/5839
